packages/ui-kit/src/components/FilterProvider/FilterProvider.tsx

```diff
@@ -11,9 +14,18 @@ export interface FilterContextValue {
   filters: FilterInputWithId[]
   /** Setter function for the Filters */
   setFilters: React.Dispatch<React.SetStateAction<FilterInputWithId[]>>
+  /** Timerange that will be provided to the child components */
+  timeRange: DateRangeOptionsProps
+  /** Setter function for the Time Range */
+  setTimeRange: React.Dispatch<React.SetStateAction<DateRangeOptionsProps>>
 }
 
-export const FilterContext = createContext<FilterContextValue>({ filters: [], setFilters: () => {} })
+export const FilterContext = createContext<FilterContextValue>({
+  filters: [],
+  setFilters: () => {},
+  timeRange: { value: '' },
```

Comment:
IMO, it is weird to have the `value` be `''`. That's not a valid time range, right? Better to set it `null` or `undefined`, IMO, rather than a sentinel value `''`. WDYT?

Reply:
`value` is the unique identifier of a time range option, not actually the value itself, I'm thinking that maybe `id` would be a different name. I agree it makes sense to make it nullable, I'll change that